.github/workflows/tests.yml

```diff
@@ -0,0 +1,40 @@
+name: tests
+
+on:
+  [push, pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-20.04
+    permissions: write-all
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.9"]
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install main dependencies
+      run: 
+        pip install -r requirements.txt
+
+    - name: Install test dependencies
+      run: 
+        pip install flake8 pytest pytest-cov
+
+    - name: Build coverage file
+      run:
+        pytest --junitxml=pytest.xml --cov-report=term-missing:skip-covered --cov=app tests/ | tee pytest-coverage.txt
+
+    - name: Pytest coverage comment
+      uses: MishaKav/pytest-coverage-comment@main
```

Comment:
Can you make sure this only comments once? Because now for every commit on an open PR, this will add a comment. Maybe you could also look into integrating PyTest running with MLLint, since I think it will show the results of that in the output

Reply:
But it's fine to make a new comment if the python files changed right?

Reply:
The problem I see with integrating pytest with mllint is that somehow the workflows need to be executed in a certain order, so that only one of them comments the "result.md" file in the PR and the workflows need to share the files between them.